src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +127,20 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void helpTest() {
```

Comment:
Let's write new tests following the new convention. Old ones can be refactored at a later time.